src/Impostor.Api/Net/Inner/RpcCalls.cs

```diff
@@ -33,5 +33,6 @@ public enum RpcCalls : byte
         RepairSystem = 28,
         SetTasks = 29,
         UpdateGameData = 30,
+        CustomRpc = byte.MaxValue
```

Comment:
This safe? I would go for `byte.MaxValue - 1` but if you checked it, it's ok.

Reply:
Didn't have any problems with this so far